### PR TITLE
Add docker and cicd

### PR DESCRIPTION
**Pull request by wanwiset25**

### CODE REVIEW

File: network.config.json [hunk 1]
```diff
@@ -1,4 +1,6 @@
 {
   "xdcparentnet": "https://devnetstats.apothem.network/devnet",
+  "xdcdevnet": "https://devnetstats.apothem.network/devnet",
```
Comment (GalaxySciTech) [inline network.config.json:4]: why there have 2 lines

Comment (wanwiset25) [reply network.config.json:4]: xdcdevnet: fixed string
xdcparentnet: the URL can be replaced during the deploy process for customization. Use devnet URL as default.

Comment (GalaxySciTech) [reply network.config.json:4]: i think if some parameters no useful , it can be deleted 

can you explain the 4 parameters work?

Comment (wanwiset25) [reply network.config.json:4]: deploy devnet
    npx hardhat run scripts/proxy/ProxyGatewayDeploy.js --network xdcdevnet
deploy testnet
    npx hardhat run scripts/proxy/ProxyGatewayDeploy.js --network xdctestnet
deploy custom url
    npx hardhat run scripts/proxy/ProxyGatewayDeploy.js --network xdcparentnet
 
The xdcdevnet and xdctestnet are fixed url not parameter. This just simplifies when you want to deploy different network. Don't need to keep editing config. Also xdcparentnet, xdcsubnet will be used in subnet generator.

**Review by GalaxySciTech — APPROVED** [review]

**Pull request merged**